db/seeds.rb

```diff
@@ -1,3 +1,8 @@
+# we use this to be able to increase the size of the seeded DB at will
+# eg.: `SEEDS_MULTIPLIER=2 rails db:seed` would double the amount of data
```

Comment:
May want to spell out the base numbers here (10 users, 25 articles)

Reply:
makes sense! I'll add it to the next PR :)